File: Python3/Alternativas/ej3.py
# -*- coding: utf-8 -*-

#Escribe un programa que lea un número e indique si es par o impar.

#1.almacenar numero (int)
#2.calcular a numero el módulo 2
#3.si el resultado es 0, numero es par
#4.si el resultado es 1, numero es impar

print("..::Calcular si un número es par o impar")
numero = int(input("Introduce un número entero> "))
if(numero%2!=0):
	print(f"{numero} es impar")
else:
	print(f"{numero} es par")

File: Java/Alternativas/ej3.java
import java.util.Scanner; //Importar la clase Scanner 

/**
 * Escribe un programa que lea un número e indique si es par o impar.
 * @author Alberto Miguel Martinez Jimenez
 */
/**
 * Algoritmo:
 * Almacenar un número (int)
 * Calcularle al número el módulo 2
 * Si es igual a cero, es par
 * Si no, es impar 
 *
 */
public class ej3 {

	public static void main(String[] args) {
		int numero;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("---Par o impar---");
		System.out.println("Introduce un número entero> ");
		numero = scanner.nextInt();
		
		if(numero%2==0) {
			System.out.println(numero+" es par");
		}else{
			System.out.println(numero+" es impar");
		}

	}

}
